nemo/collections/llm/modelopt/quantization/quantizer.py

```diff
@@ -46,6 +46,7 @@
         "int8": mtq.INT8_DEFAULT_CFG,
```

Comment:
I wonder if we could define `QUANT_CFG_CHOICES` somewhere in modelopt and just import it here.

The reason are to keep the choices up-to-date in the source library, and also make the code here in NeMo functional regardless which modelopt version is used (with this change here, this code will work only for 0.27.0 while in fact it's at least modelopt >= 0.23.2 <= 0.27.0) to make it more robust.